src/app/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Typography } from '@mui/material/';
+import CartItem from './CartItem/CartItem';
+
+const Cart: React.FC = () => {
+  const productsMock: {
+    id: number;
+    title: string;
+    price: number;
+    quantity: number;
+  }[] = [
+    { id: 1, title: 'Product 1', price: 10, quantity: 1 },
+    { id: 2, title: 'Product 2', price: 20, quantity: 1 },
+    { id: 3, title: 'Product 3', price: 30, quantity: 1 },
+  ];
+
+  return (
+    <Box>
+      {productsMock.map((product, index) => (
+        <CartItem key={index} product={product} />
```

Comment:
uważaj na to - bo rzeczy w koszyku będą się modyfikować więc raczej key={product.id}